common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -171,6 +171,8 @@
   "feature.wynntils.gammabright.lastGamma.name": "Last Gamma",
   "feature.wynntils.gammabright.name": "Gammabright Keybind",
   "feature.wynntils.gearViewer.name": "Gear Viewer",
+  "feature.wynntils.getCombatXPGainMessages.description": "Do you want to see updates to your XP as overlay messages?",
+  "feature.wynntils.getCombatXPGainMessages.name": "Combat XP Gain Messages",
```

Comment:
So you still don't use the run configurations in your IDE. Or you just didn't test this. But, you didn't add an i18n for the feature name, which should make it fail immediately in development environment.

Reply:
In fact, the i18n for the variable, which is not needed as I commented above, should not even look like this. You missed the whole feature part.